src/Symfony/Component/Messenger/Bridge/Redis/composer.json

```diff
@@ -17,12 +17,13 @@
     ],
     "require": {
         "php": ">=8.0.2",
+        "ext-redis": "*",
```

Comment:
maybe use `>=4.5` here ?

Reply:
Can composer really see the version of extensions?

I didnt know that.

Reply:
yes it does (thanks to `phpversion('redis')`). For core extensions, it does not make sense (in modern PHP versions, it reports the version of PHP. For older PHP versions, it was reporting garbage). But for pecl extensions, the version makes sense.